packages/components/src/internal/components/domainproperties/SystemFields.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FC, memo, useCallback, useState } from 'react';
+
+import classNames from 'classnames';
+
+import { Collapse } from 'react-bootstrap';
+
+import { Grid } from '../base/Grid';
+
+import { SystemField } from './models';
+import { EXPAND_TRANSITION } from './constants';
+
+interface Props {
+    systemFields: SystemField[];
```

Comment:
minor: the const name is `SystemFields` so I would probably just call this prop `fields`